llvm/test/tools/llvm-profdata/memprof-merge-v1.test

```diff
@@ -0,0 +1,19 @@
+REQUIRES: x86_64-linux
+
+RUN: echo ":ir" > %t.proftext
+RUN: echo "main" >> %t.proftext
+RUN: echo "742261418966908927" >> %t.proftext
+RUN: echo "1" >> %t.proftext
+RUN: echo "1" >> %t.proftext
+
+To update the inputs used below run Inputs/update_memprof_inputs.sh /path/to/updated/clang
+RUN: llvm-profdata merge %t.proftext %p/Inputs/basic.memprofraw --memprof-version=1 --profiled-binary %p/Inputs/basic.memprofexe -o %t.prof
```

Comment:
Since afaict this test is identical to the v0 version other than this RUN line - can you combine them and do 2 RUN invocations?

Reply:
Yes, I just merged the two tests into one file in the latest iteration.